src/extensions/default/HealthData/HealthDataManager.js

```diff
@@ -306,25 +308,41 @@ define(function (require, exports, module) {
     function checkAnalyticsDataSend(event, Eventparams, forceSend) {
         var result         = new $.Deferred(),
             isHDTracking   = prefs.get("healthDataTracking"),
-            isEventDataAlreadySent;
+            isEventDataAlreadySent,
+            url = brackets.config.healthDataServerURL,
+            sendFailed = false;
 
         var options = {
             location: {
                 scope: "default"
             }
         };
 
+        var handleFail = function handleFail() {
```

Comment:
Probably you wrote this syntax by mistake. This is useful when we are running in IE or when we want to assign a small function name to use inside the function.

Reply:
It's not a good idea to write nested function if you function executed multiple time.